packages/map/src/CdcMap.js

```diff
@@ -1533,10 +1533,10 @@ const CdcMap = ({ className, config, navigationHandler: customNavigationHandler,
             className={tooltips.capitalizeLabels ? 'capitalize tooltip' : 'tooltip'}
           />
         )}
-        {state.general.title &&
+        {(state.general.title || general.superTitle) &&
           <header className={general.showTitle === true ? 'visible' : 'hidden'} {...(!general.showTitle || !state.general.title ? { 'aria-hidden': true } : { 'aria-hidden': false })}>
             <div role="heading" className={'map-title ' + general.headerColor} tabIndex="0" aria-level="2">
```

Comment:
@ShawnStringfield - 

- Move aria-level and role to the div on title & verify that looks good.